#! /bin/sh /usr/share/dpatch/dpatch-run
## 00-remove-local-gdata.dpath.dpatch by  <julien.lavergne@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use packaged version of gdata and atom instead of local files.

@DPATCH@
diff -urNad conduit-0.3.16~/conduit/modules/GoogleModule/GoogleModule.py conduit-0.3.16/conduit/modules/GoogleModule/GoogleModule.py
--- conduit-0.3.16~/conduit/modules/GoogleModule/GoogleModule.py	2009-05-20 08:30:07.000000000 +0200
+++ conduit-0.3.16/conduit/modules/GoogleModule/GoogleModule.py	2009-06-27 17:59:42.000000000 +0200
@@ -26,7 +26,7 @@
 
 #Distributors, if you ship python gdata >= 1.0.10 then remove this line
 #and the appropriate directories
-Utils.dataprovider_add_dir_to_path(__file__)
+#Utils.dataprovider_add_dir_to_path(__file__)
 try:
     import atom.service
     import gdata.service
